app/Http/Controllers/UserController.php

```diff
@@ -128,10 +130,9 @@ public function getUsers(User $user, Request $req) {
         return response(json_encode($toReturn), 200);
     }
 
-    public function getUser(User $user) {
-        $id = Input::get('id');
-        $user = $user->with('antenna')->findOrFail($id);
+    public function getUser(Request $req, User $user, RolesRepository $repo) {
 
+        dd($repo->getRoles($req, $user));
```

Comment:
Why commit this? This just killed the whole method.
Please, only push changes that DO NOT break the existing things.
Thanks